integr8ly/tutorial-web-app-operator
# INTLY-3286: Handle reconcile of installed_service env var

## What
Adds a parameter to the webapp template for `INSTALLED_SERVICES`. This var will contain an object that has information of the services installed by Integreatly on OS4. This should be added as an env var so that the webapp has access to this information.
- [x] Allows param `INSTALLED_SERVICES` to be specified in the deployment config env var. This is added to the deployment config if defined in the webapp custom resource.
- [x] Bump operator version to `0.0.31`

## Verification Steps
1. Run the operator
2. Create a webapp CR with the following param defined in `spec.template.parameters`:
   ```
    INSTALLED_SERVICES: |-
    { 
      "3scale":{ 
          "Host":"https://3scale.example.com",
          "Version":"0.0.1"
      },
      "codeready":{ 
          "Host":"https://codeready.example.com",
          "Version":"0.0.1"
      },
      "fuse-managed":{ 
          "Host":"https://syndesis.example.com",
          "Version":"0.0.1"
      },
      "launcher":{ 
          "Host":"https://launcher.example.com",
          "Version":"0.0.1"
      },
      "user-rhsso":{ 
          "Host":"https://sso.example.com",
          "Version":"0.0.1"
      }
    }
3. **Verify**: `INSTALLED_SERVICES` is added as an env var to the deployment config and webapp pod

**Related PR**: 
- https://github.com/integr8ly/tutorial-web-app/pull/512
- https://github.com/integr8ly/integreatly-operator/pull/125
